test/xspec.bats

```diff
@@ -326,3 +326,20 @@ teardown() {
 	grep '<style type="text/css">' ../tutorial/xspec/escape-for-regex-result.html
 	grep 'margin-right:' ../tutorial/xspec/escape-for-regex-result.html
 }
+
+
+@test "running XSpec via ant for XSLT support" {
+    run ant -buildfile ${PWD}/../build.xml -Dxspec.xml=${PWD}/../tutorial/escape-for-regex.xspec -lib ${SAXON_CP} -lib /tmp/ant/lib 
```

Comment:
If I recall correctly, `-Dclean.output.dir=true` is necessary here too.

Reply:
Good catch! I added it in https://github.com/xspec/xspec/commit/ab86e9fb7dcc1e4c56b6f12bab57447e586df5bb